integration_tests/models/expected/snowplow_web_consent_totals_expected_stg.sql

```diff
@@ -0,0 +1,21 @@
+select
+
+  replace(cast(consent_version as {{ dbt.type_string() }}), '.0', '') ||'.0' as consent_version,
```

Comment:
Same as above, type casting did not work in the seed directly, the sample data is sent in such a format that it has .0 at the end. I treat that field as a string but then the .0 disappears along the way as I'm casting to string. I strip out the .05 because in some data warehouses it does not behave like this and this is to make sure that we have an even finish before I readd the .05 in the end. Not sure if I explained clearly but that's what's behind. :)